### PR TITLE
added a cross reference about correlating logs.

### PR DESCRIPTION
**Pull request by zippy1981**
### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -377,7 +377,8 @@ Sets the URL endpoint where traces are sent. Overrides `DD_AGENT_HOST` and `DD_T
 
 `DD_LOGS_INJECTION`
 : **TracerSettings property**: `LogsInjectionEnabled` <br>
-Enables or disables automatic injection of correlation identifiers into application logs.
+Enables or disables automatic injection of correlation identifiers into application logs.<br/>
+Note that your logger needs to have a `source` that set to the `trace_id` mapping properly. The default source for .NET Applications, `csharp` does this automatically. For more information [see the FAQ on correlation troubleshooting](/tracing/faq/why-cant-i-see-my-correlated-logs-in-the-trace-id-panel.md#trace_id-option).
```
Comment (maycmlee) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:381]:
```suggestion
Your logger needs to have a `source` that sets the `trace_id` mapping correctly. The default source for .NET Applications, `csharp`, does this automatically. For more information [see the FAQ on correlation troubleshooting](/tracing/faq/why-cant-i-see-my-correlated-logs-in-the-trace-id-panel.md#trace_id-option).
```

Comment (maycmlee) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:381]:
```suggestion
Your logger needs to have a `source` that sets the `trace_id` mapping correctly. The default source for .NET Applications, `csharp`, does this automatically. For more information [see the FAQ on correlation troubleshooting](/tracing/faq/why-cant-i-see-my-correlated-logs-in-the-trace-id-panel.md#trace_id-option).
```

**Review by maycmlee — CHANGES_REQUESTED** [review]

**Comment by zippy1981** [comment]
@maycmlee I made the requested changes. Can this be approved?

**Review by zacharycmontoya — APPROVED** [review]
LGTM from the engineering side

**Review by maycmlee — APPROVED** [review]

**Review by zacharycmontoya — CHANGES_REQUESTED** [review]
Sorry, before merging can these changes be copied over to the `dotnet-framework.md` file so the two files are in-sync?

**Comment by zippy1981** [comment]
Done @zacharycmontoya

**Review by zacharycmontoya — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**